library/mono

```diff
@@ -23,4 +21,13 @@ onbuild: git://github.com/mono/docker@adc7a3ec47f7d590f75a4dec0203a2103daf8db0 3
 3.8.0-onbuild: git://github.com/mono/docker@adc7a3ec47f7d590f75a4dec0203a2103daf8db0 3.8.0/onbuild
 3.8-onbuild: git://github.com/mono/docker@adc7a3ec47f7d590f75a4dec0203a2103daf8db0 3.8.0/onbuild
 
-alpha: git://github.com/mono/docker@f639c9ba71c3e3601c079b353e12351bd008b11e alpha
+4.0.1: git://github.com/mono/docker@0d3556995aa47043059d60c42321e8ccaf173363 4.0.0
+4.0.0: git://github.com/mono/docker@0d3556995aa47043059d60c42321e8ccaf173363 4.0.0
```

Comment:
If you're open to it, I'd be happy to make a PR so that what I mean is more clear.  We usually also include a script to generate this library file so that the future PRs to this repo are easier (see https://github.com/docker-library/percona/blob/master/generate-stackbrew-library.sh for one specific example of that).